pkg/util/aws.go

```diff
@@ -87,7 +87,7 @@ func AttachAWSCredentials(cfg *aws.Config, profile *AWSProfile) error {
 
 func GetQueueURL(cp client.ConfigProvider, queueName string, awsRegion string) string {
 	stsClient := sts.New(cp)
-	identity, err := stsClient.GetCallerIdentity(nil)
+	identity, err := stsClient.GetCallerIdentity(&sts.GetCallerIdentityInput{})
```

Comment:
What is the actual error? This is the first time that I see this would actually be necessary.